geometry_msgs/Twist twist
geometry_msgs/Pose pose
float32 propeller_mass

